### ISSUE TITLE
Tables are too wide, causing horizontal scroll-bars

### ISSUE THREAD

**Issue by ashb** [apache/airflow-site#260]
The table for extra packages is too wide and has a horizontal scroll bar

![image](https://user-images.githubusercontent.com/34150/78890621-df973f80-7a5d-11ea-8718-43831036648a.png)

**Comment by ashb** [comment]
@somraj-2024 One thing to bear in mind is that the correct fix will involve a fix to the sphinx docs theme as one PR, and then once that is merged we can make a separate PR to update all the existing rendered docs.

**Comment by potiuk** [comment]
Well. With the current build setup it will need at least 2 PRs and releasing theme to PyPI: 

a) update to sphinx theme with PR
b) releasing the theme in PyPI (i.e. maintainer) 
c) rebuilding the docs to see if it works

Hopefully https://github.com/apache/airflow/issues/45613#issue-2783728135 will solve this complexity (when we get to it).

**Comment by somraj-2024** [comment]
@potiuk I have fixed it and raised a PR. Kindly verify and do let me know if I missed out something. And if everything is ok, kindly merge the PR.

**Comment by prashant2007-wq** [comment]
Hi maintainers,

I would like to work on this issue as my first contribution to Apache Airflow Site.

I will investigate the table overflow issue, reproduce the horizontal scrollbar behavior, and then make a focused CSS fix so wide tables remain readable and responsive without breaking the existing documentation layout.

My plan is to keep the change minimal, test the affected documentation pages locally, and ensure the fix works on smaller screen sizes as well.

Please let me know if this issue is still available to work on.

**Comment by potiuk** [comment]
check and see, we do not assign people but if you can see that you can improve things and make PR showing before /after that is more than enough,
